### ISSUE TITLE
handling all-zero objective in writeLp

### ISSUE THREAD

**Issue by svigerske** [coin-or/CoinUtils#74]
Issue created by migration from Trac.

Original creator: fmargot

Original creation time: 2012-01-01 11:23:34

Version: 

CC:  fmargot@andrew.cmu.edu

Line 817 of CoinLpIO.cpp should be changed from

```
   if(cnt_print % numberAcross != 0) {
```

to
```
   if((cnt_print == 0) || (cnt_print % numberAcross != 0)) {
```
in order to handle correctly the case where the objective function is all-zero.

**Comment by svigerske** [comment]
Comment by fmargot created at 2012-01-02 07:38:40

.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-04-09 18:45:14

Committed this fix with 6ab3f53ca1.

This ticket can be closed.

**Comment by svigerske** [comment]
Comment by @tkralphs created at 2013-07-11 14:28:11

Resolution: fixed
